readme.md

```diff
@@ -81,4 +70,443 @@ Each of the above sections has its own `.rest` file in the `requests` directory
+    "$shared": {
+      "flagsUrl": "https://api.optimizely.com/flags/v1",
+      "baseUrl": "https://api.optimizely.com/v2",
+      "token": "Bearer {{yourToken}}",
+      "projectId": {{yourProjectId}},
+      "flagKey": {{your_flag_key}},
++     "environment": "development",
++     "eventId": 27280660613
+    }
+  }
+}
+```
+Now we will execute the **6_create_experiment.rest**. You should see a response like this:
+````json
+{
+  "url": "/projects/{{your_project_id}}/flags/inventory_on_pdp/environments/development/ruleset",
+  "update_url": "/projects/{{your_project_id}}/flags/inventory_on_pdp/environments/development/ruleset",
+  "enable_url": "/projects/{{your_project_id}}/flags/inventory_on_pdp/environments/development/ruleset/enabled",
+  "rules": {
+    "stock_message_test": {
+      "key": "stock_message_test",
+      "name": "Stock Message Test",
+      "description": "",
+      "variations": {
+        "off": {
+          "key": "off",
+          "name": "Off",
+          "percentage_included": 3333
+        },
+        "on_show_amounts": {
+          "key": "on_show_amounts",
+          "name": "On Show Amounts",
+          "percentage_included": 3333
+        },
+        "on_show_amounts_red": {
+          "key": "on_show_amounts_red",
+          "name": "On Show Amounts Red",
+          "percentage_included": 3334
+        }
+      },
+      "type": "a/b",
+      "distribution_mode": "manual",
+      "id": 295492,
+      "urn": "rules.flags.optimizely.com::295492",
+      "archived": false,
+      "enabled": false,
+      "created_time": "2024-01-29T08:54:17.963801Z",
+      "updated_time": "2024-01-29T08:54:17.963808Z",
+      "audience_conditions": [],
+      "audience_ids": [],
+      "percentage_included": 10000,
+      "metrics": [
+        {
+          "event_id": {{your_event_id}},
+          "event_type": "custom",
+          "scope": "visitor",
+          "aggregator": "unique",
+          "winning_direction": "increasing",
+          "display_title": "add_to_cart"
+        }
+      ],
+      "allow_list": {},
+      "group_rule": {
+        "group_id": 0,
+        "traffic_allocation": 0
+      },
+      "group_remaining_traffic_allocation": 100
+    }
+  },
+  "rule_priorities": [
+    "stock_message_test"
+  ],
+  "id": 323190,
+  "urn": "rulesets.flags.optimizely.com::323190",
+  "archived": false,
+  "enabled": false,
+  "updated_time": "2024-01-29T08:54:18.279750Z",
+  "flag_key": "inventory_on_pdp",
+  "environment_key": "development",
+  "environment_name": "Development",
+  "default_variation_key": "off",
+  "default_variation_name": "Off",
+  "revision": 2
+}
+
+````
+Congratulations! You just created your first experiment.
+
+### 7. Launch the experiment.
+Now we will launch the experiment we just created. Execute the endpoints in **7_launch_experiment.rest**.  
+Sample response:
+```json
+{
+  "url": "/projects/{{your_project_id}}/flags/inventory_on_pdp/environments/development/ruleset",
```

Comment:
nit: consider adding parameter for environment in the endpoints for the rule.

Reply:
Okay. I'll update it and push the new changes.